chaos/default_settings.py

```diff
@@ -6,7 +6,7 @@
 
 DEBUG = (os.getenv('DEBUG', 1) == 1)
 
-NAVITIA_URL = str(os.getenv('NAVITIA_URL', 'http://navitia2-ws.ctp.dev.canaltp.fr'))
+NAVITIA_URL = str(os.getenv('NAVITIA_URL', 'https://navitia2-ws.ctp.dev.canaltp.fr'))
```

Comment:
```suggestion
NAVITIA_URL = str(os.getenv('NAVITIA_URL', 'http://navitia2-ws.ctp.dev.canaltp.fr'))
```
https doesn't exist (and will perhaps will never be as its DEV)